composer.json

```diff
@@ -26,8 +26,7 @@
         "ext-json": "*",
         "ext-sqlite3": "*",
         "acquia/coding-standards": "^0.5.0",
-        "composer/composer": "^1.7",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.5",
```

Comment:
I don't think removing dealerdircet/phpcodesniffer-composer-installer is necessary. Version 0.7 support Composer 2: https://github.com/Dealerdirect/phpcodesniffer-composer-installer/releases/tag/v0.7.0